livedata/src/main/java/io/getstream/chat/android/livedata/controller/ChannelControllerImpl.kt

```diff
@@ -691,7 +692,7 @@ class ChannelControllerImpl(
 
         // update all the fresh messages
         for (message in freshMessages) {
-            copy[message.id] = message
+            copy[message.id] = message.copy()
```

Comment:
Because we have a mutable model and we are not updating anything, I think it is not needed to call `.copy()` method here